turbo/engineapi/engine_types/status.go

```diff
@@ -5,6 +5,7 @@ type EngineStatus string
 const (
 	ValidStatus            EngineStatus = "VALID"
 	InvalidStatus          EngineStatus = "INVALID"
+	InvalidParamsError     EngineStatus = "INVALID_PARAMS_ERROR"
```

Comment:
I don't think we need this addition. `InvalidParamsError` is already defined in rpc/errors.go. It's not one of the [valid status values](https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#payloadstatusv1).